docs/source/deployorderer/ordererplan.md

```diff
@@ -0,0 +1,84 @@
+# Planning for an ordering service
+
+Audience: Architects, network operators, users setting up a production Fabric network who are familiar with Transport Layer Security (TLS), Public Key Infrastructure (PKI) and Membership Service Providers (MSPs).
+
+Check out the conceptual topic on [The Ordering Service](../orderer/ordering_service.html) for an overview on ordering service concepts, implementations, and the role an ordering service plays in a transaction.
+
+In a Hyperledger Fabric network, a node or collection of nodes together form what's called an "ordering service", which literally orders transactions into blocks that are processed peers and committed to their ledgers. This separates Fabric from other distributed blockchains, such as Ethereum and Bitcoin, in which this ordering is done by any and all nodes, requiring the establishment of probabilistic consensus algorithms which eventually guarantee ledger consistency to a high degree of probability, but which are still vulnerable to divergent ledgers (also known as a ledger “fork”), where different participants in the network have a different view of the accepted order of transactions.
```

Comment:
Beyond the suggestion I'm making in the first sentence, the last sentence is still too long and complex to understand. I'm not understanding the point you are trying to make... whether peer nodes participate in ordering or not has nothing to do with probabilistic consensus and potential forks. I think the entire last sentence could be removed.

```suggestion
In a Hyperledger Fabric network, a node or collection of nodes together form what's called an "ordering service", which literally orders transactions into blocks, which peers will then validate and commit to their ledgers. This separates Fabric from other distributed blockchains, such as Ethereum and Bitcoin, in which this ordering is done by any and all nodes, requiring the establishment of probabilistic consensus algorithms which eventually guarantee ledger consistency to a high degree of probability, but which are still vulnerable to divergent ledgers (also known as a ledger “fork”), where different participants in the network have a different view of the accepted order of transactions.
```

Reply:
I think it's important for new users to understand that one of the key differences between Fabric and other blockchain networks is that the ordering process is performed by a dedicated node. This language is largely copied from the ordering service key concept doc. I think it's probably fine to stop at "any and all nodes", though.